blazar/manager/enforcement.py

```diff
@@ -380,7 +392,8 @@ def check_usage_against_allocation_pre_update(self, reservation_values,
                 raise BillingError(
                     'Reservation update would spend {:.2f} more SUs, only '
                     '{:.2f} left'.format(estimated_requested, left))
-        except redis.exceptions.ConnectionError:
+        except (redis.exceptions.ConnectionError,
```

Comment:
This all looks good, but it also make me notice how redundant the try-except blocks are in the check_usage* functions. There are already the same try-except blocks in get_balance, get_encumbered, increase_encumbered, etc... That's a lot of redundant code that can be eliminated. Maybe a decorator function would be useful around all these. Not necessary to change for this, but figured I'd point that out.

Reply:
I had the same thought :) - maybe I can throw something together quickly.